test/Flick/AnimatedCarousalPage.xaml.cs

```diff
@@ -71,42 +149,91 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
+             * control so the user can set it to any value */
+            scaleYExpression.SetScalarParameter("scaleRatioXY", (float)ItemScaleRatio);
+            scaleYExpression.SetScalarParameter("spacing", (float)layout.Spacing);
+
+            // scale the item based on the distance of the item relative to the center of the viewport.
+            scaleYExpression.Expression = scaleExpressionString;
+
+            // item.StartAnimation("Scale.Y", scaleXExpression);
+            scaleYExpression.Target = "Scale.Y";
+            animationGroup.Add(scaleYExpression);
+
+            /* TODO: Create an ExpressionAnimation to be applied to each item's Offset property that will 
+             * allow each item to maintain the fixed Layout.Spacing even when a Scale animation has been applied */
+            var offsetExpression = scrollProperties.Compositor.CreateExpressionAnimation();
+            offsetExpression.SetReferenceParameter("svVisual", svVisual);
+            offsetExpression.SetReferenceParameter("scrollProperties", scrollProperties);
+            offsetExpression.SetReferenceParameter("item", item);
+            offsetExpression.SetScalarParameter("itemScaleRatio", (float)ItemScaleRatio);
+            offsetExpression.SetScalarParameter("spacing", (float)layout.Spacing);
+            //offsetExpression.Expression = "Vector3(0,200,0)";
+            //offsetExpression.Expression = "Vector3(((((item.Offset.X + (item.Size.X/2)) < ((svVisual.Size.X/2) - scrollProperties.Translation.X)) ? 1 : -1) * (item.Size.X * (1 - clamp((scaleRatioXY * (1 + (1 - (abs((item.Offset.X + (item.Size.X/2)) - ((svVisual.Size.X/2) - scrollProperties.Translation.X)) / (item.Size.X + spacing))))), scaleRatioXY, 1)) / 2)), 0, 0)";
+            //offsetExpression.Expression = "Vector3(((abs((item.Offset.X + (item.Size.X/2)) - ((svVisual.Size.X/2) - scrollProperties.Translation.X)) > (item.Size.X + spacing)) ? ((((item.Offset.X + (item.Size.X/2)) < ((svVisual.Size.X/2) - scrollProperties.Translation.X)) ? 1 : -1) * min((((item.Size.X * (1 - itemScaleRatio)) * ((abs((item.Offset.X + (item.Size.X/2)) - ((svVisual.Size.X/2) - scrollProperties.Translation.X))) / (item.Size.X + spacing))) - (item.Size.X + spacing)), 0)) : 0), 0, 0)";
+            //item.StartAnimation("Offset", offsetExpression);
+            offsetExpression.Target = "Translation";
+            //offsetExpression.Target = "Offset.Y";
+            //args.Element.StartAnimation(offsetExpression);
+            //animationGroup.Add(offsetExpression);
+
+            item.StartAnimationGroup(animationGroup);
         }
 
-        private void OnItemClicked(object sender, RoutedEventArgs e)
+        private void OnItemGotFocus(object sender, RoutedEventArgs e)
         {
-            ScrollToCenterOfViewport(sender);
-            //sv.ChangeView((layout.ItemWidth + layout.Spacing) * 500, null, null);
+            //ScrollToCenterOfViewport(sender);
```

Comment:
dont want this behavior ? If so, can you remove the method ?

Reply:
Done.